DataFormats/Detectors/TOF/include/DataFormatsTOF/CalibTimeSlewingParamTOF.h

```diff
@@ -73,8 +80,8 @@ class CalibTimeSlewingParamTOF
  private:
   // TOF channel calibrations
   int mChannelStart[NSECTORS][NCHANNELXSECTOR];           ///< array with the index of the first element of a channel in the time slewing vector (per sector)
-  std::vector<std::pair<float, float>>* mTimeSlewing[18]; //! pointers to the sector vectors
-
+  std::vector<std::pair<unsigned short, short>> mTimeSlewing[18]; ///< array of sector vectors; first element of the pair is TOT (in ps), second is t-texp_pi (in ps)
+  /*
```

Comment:
Since these commented line will never be used, could you suppress them?

Reply:
Done